### ISSUE TITLE
Removing last vertex causes BoundsError

### ISSUE THREAD

**Issue by epatters** [sbromberger/MetaGraphs.jl#33]
Removing the last vertex in a `MetaDiGraph` causes a `BoundsError`:

```
julia> g = MetaDiGraph(); add_vertices!(g, 3); rem_vertex!(g, 3)
ERROR: BoundsError: attempt to access 2-element Array{Array{Int64,1},1} at index [3]
Stacktrace:
 [1] rem_vertex!(::MetaGraphs.MetaDiGraph{Int64,Float64}, ::Int64) at /Users/epatters/.julia/v0.6/MetaGraphs/src/MetaGraphs.jl:154
````

**Comment by sbromberger** [comment]
Thanks for reporting this. Fixed via #34.

**Comment by epatters** [comment]
Thanks for the quick fix!
